README.md

```diff
@@ -1,33 +1,108 @@
+
+Boot from the USB stick.
+
+The instalation will start and take a few minutes. After that the system will shutdown.
+
+Remove the USB stick and restart the system.
+
+The deployment will start. This takes some time and the system will shutdown when the deployment is finished.
+This step can take at least one hour or more.
+
+System is ready for use, by default DHCP is tried on the first network device.
+
+Login via ssh. Use the user **dragon** with the password **password**.
+
+```bash
+ssh dragon@IpFromYourServer
+```
+
+Copy the `/home/dragon/wireguard-client.conf` file to your device. This is necessary for using the web endpoints on your device.
+Rename the wireguard config file to something like `ciab.conf`.
+> **Note:**
+> Be aware that the name does not contain any special characters!
+
+```bash
+scp dragon@IpFromYourServer:/home/dragon/wireguard-client.conf /home/ubuntu/ciab.conf
+```
+
+Install wireguard on your device, if you have not done this before. For instructions how to do it on your device, please have
+a look on the documentation of your used distribution.
+
+Start the wireguard tunnel.
+
+```bash
+wg-quick up /home/ubunu/ciab.conf
+```
+
+Now your Cloud in a Box is up and usable via the following URLs:
+
+| Name              | URL                                           | Username  | Password                                 |
+|-------------------|-----------------------------------------------|-----------|------------------------------------------|
+| ARA               | <https://ara.services.in-a-box.cloud>         | ara       | password                                 |
+| Flower            | <https://flower.services.in-a-box.cloud>      | -         | -                                        |
+| Grafana           | <https://api.in-a-box.cloud:3000>             | admin     | password                                 |
+| Homer             | <https://homer.services.in-a-box.cloud>       | -         | -                                        |
+| Horizon Admin     | <https://api.in-a-box.cloud>                  | admin     | password                                 |
+| Horizon Test-User | <https://api.in-a-box.cloud>                  | test      | test # FIX ME                            |
+| Kibana            | <https://api.in-a-box.cloud:5601>             | kibana    | password                                 |
+| Netbox            | <https://netbox.services.in-a-box.cloud>      | admin     | password                                 |
+| Netdata           | <http://manager.systems.in-a-box.cloud:19999> | -         | -                                        |
+| phpMyAdmin        | <https://phpmyadmin.services.in-a-box.cloud>  | root      | password                                 |
+| RabbitMQ          | <https://api.in-a-box.cloud:15672>            | openstack | BO6yGAAq9eqA7IKqeBdtAEO7aJuNu4zfbhtnRo8Y |
+
+> **Note:**
+> Watch out for http and https. It is necessary to use the right one.
+
```

Comment:
Is it worth mentioning that most web interfaces are linked from homer?
So homer becomes the landing page / dashboard for all of these.